### PR TITLE
Simulator token and type selector

### PR DESCRIPTION
**Pull request by tiagofilipenunes**

### CODE REVIEW

File: src/components/simulator/SimulatorStrategyType.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+interface Props {
+  strategyType: SimulatorType;
+}
+
+export const SimulatorStrategyType: FC<Props> = ({ strategyType }) => {
+  const list = useRef<HTMLUListElement>(null);
+  const navigate = useNavigate();
+
+  const items: {
```
Comment (pingustar) [inline src/components/simulator/SimulatorStrategyType.tsx:17]: lets declare a local interface for readability

File: src/components/simulator/SimulatorStrategyType.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      </header>
+      <ul
+        onKeyDown={handleKeyDown}
+        ref={list}
+        role="tablist"
+        className={`grid grid-cols-2 gap-8`}
+      >
+        {items.map(({ label, svg, tooltipText }) => (
+          <li role="none" key={label} className="relative">
+            <button
```
Comment (pingustar) [inline src/components/simulator/SimulatorStrategyType.tsx:65]: I would probably use a <Link> component here. Reason is, that its just a simple navigation and you benefit from stuff like right-click it and open in new tab.

**Review by pingustar — APPROVED** [review]
nice work!

File: src/components/simulator/SimulatorStrategyType.tsx (hunk 1 above)
Comment (tiagofilipenunes) [reply src/components/simulator/SimulatorStrategyType.tsx]: done

File: src/components/simulator/SimulatorStrategyType.tsx (hunk 2 above)
Comment (tiagofilipenunes) [reply src/components/simulator/SimulatorStrategyType.tsx]: done

**Pull request merged**